solutions/quickstart/variables.tf

```diff
@@ -158,6 +166,18 @@ variable "addons" {
       version = "416"
```

Comment:
Please use a newer addon version to deploy with OCP 4.19 (which is now the default version)